charts/openmetadata/values.yaml

```diff
@@ -389,5 +389,11 @@ startupProbe:
     path: /healthcheck
     port: http-admin
 
+podDisruptionBudget:
+  enabled: true
+  config:
+    maxUnavailable: 1
+    # minAvailable: 1
```

Comment:
This should not be commented out as it will be a required property when enabled right ?

Reply:
Correct, un-commenting this one in the following commit.